readme.md

```diff
@@ -185,6 +186,18 @@ got('todomvc.com', {
 ```
 
 
+## Node 0.10
+
+It is a known issue with Node [http.Agent](https://nodejs.org/docs/v0.10.39/api/http.html#http_class_http_agent) and `agent.maxSockets`, which is set to `5`. This can cause low performance of application and (in rare cases) deadlocks. To avoid this you can set it manually:
```

Comment:
Fixed a little bit - but still can be wrong/confusing.

p.s. This agent tuning does not affects compatibility (it will work on 0.10) - but, in some cases, performance.